docs/development/plugin-triggers.md

```diff
@@ -866,6 +866,23 @@ APP="$1"; IMAGE_TAG="$2"
 some code to remove a docker hub tag because it's not implemented in the CLI....
 ```
 
+### `uninstall`
+ - Description: Used to cleanup after itself.
```

Comment:
Newline before this one. Also add a warning to check the plugin name or people are going to do silly stuff and blame Dokku when their app's break.

Also, add the following to the commit to skip tests since its just a documentation change:

```
[ci skip]
```